tools/benchmark.sh

```diff
@@ -48,6 +48,9 @@ mkdir -p $OUTDIR
 export BENCHMARK_TO_FILE="--benchmark_format=csv :>$OUTDIR/$NODE-benchmark.csv"
 make run-benchmark/fast
 
+du $SCRIPT_DIR/../build/*/*/osqueryi | awk '{print "\"EXECUTABLE_osqueryi_size\","$1",,,,,"}' >>$OUTDIR/$NODE-benchmark.csv
```

Comment:
Could be useful to copy the binaries to `/tmp` then `strip` them. It'll be somewhat relative, but we do ship the stripped versions in packages. :)